build/ghost-buster/Dockerfile

```diff
@@ -1,44 +1,57 @@
-FROM node:0.12
+FROM alpine:latest
```

Comment:
it shouldn't use latest for alpine. Change from 3.3 to 3.4 proved this is not a good idea. Several things changed and some in odd an unexpected ways. So, 3.4, not latest pls.